src/test/java/tests/RegisterTests.java

```diff
@@ -0,0 +1,27 @@
+package tests;
+
+import helpers.DataHelper;
+import helpers.LogHelper;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import page_objects.RegisterPage;
+
+public class RegisterTests extends BaseTests {
+
+    RegisterPage registerPage = new RegisterPage();
+    private final String email = DataHelper.getRandomEmail();
+    private final String password = DataHelper.getRandomPassword();
+    private final String pid = DataHelper.getRandomNumber();
```

Comment:
no need to random password

Reply:
I removed it successfully. Help me check it again